tests/config/pytest_software.yml

```diff
@@ -71,6 +76,14 @@ bismark_methylation_extractor:
   - software/bismark/genome_preparation/**
   - tests/software/bismark/methylation_extractor/**
 
+bismark_report:
```

Comment:
Why are there all these unrelated additions and deletions in this file?

Reply:
I'm not sure I used `nf-core modules create . --tool prokka --author @rpetit3 --label prokka --meta` to create to module.

Before merging, I'll bringing the lates

Reply:
ok, maybe a `git rebase master` will do then.